server/services/ciagInductionService.ts

```diff
@@ -13,7 +13,6 @@ export default class CiagInductionService {
       const ciagInduction = await this.getCiagInduction(prisonNumber, token)
       return toWorkAndInterests(ciagInduction)
     } catch (error) {
-      logger.error(`Error retrieving data from CIAG Induction API: ${JSON.stringify(error)}`)
```

Comment:
I moved the log messages down into the private function; otherwise we would have had 3 occurrences of this same error logging (IE. I would have added it to the new `ciagInductionExists` function otherwise)